### PR TITLE
[Prow] Add manual trigger job step in README

### PR DESCRIPTION
**Pull request by yutongz**
**Release note**:

```release-note
none
```

### CODE REVIEW

File: prow/README.md [hunk 1]
```diff
@@ -155,3 +155,28 @@ $ chmod +x prow/my-presubmit.sh
 This repository (istio/test-infra) also provides Prow jobs.
 
 - `test-infra-presubmit` - run the linting and testing
+
+### Manually Trigger a Prow Job
+
+
+> **Never do this unless necessary AND you are authorized by istio EngProd team (istio-engprod@google.com)**
```
Comment (sebastienvas) [inline prow/README.md]: Please remove the istio-engprod as this is google internal, and use the slack channel instead.

File: prow/README.md [hunk 2]
```diff
@@ -155,3 +155,28 @@ $ chmod +x prow/my-presubmit.sh
...
 
 - `test-infra-presubmit` - run the linting and testing
+
+### Manually Trigger a Prow Job
+
+
+> **Never do this unless necessary AND you are authorized by istio EngProd team (istio.slack.com test-infra channel)**
+
+
+Connect to Prow cluster and then:
```
Comment (chxchx) [inline prow/README.md:165]: Explain how to connect to a prow cluster

File: prow/README.md [hunk 3]
```diff
@@ -155,3 +155,28 @@ $ chmod +x prow/my-presubmit.sh
...
+```bash
+$ git clone https://github.com/kubernetes/test-infra
+$ bazel build //prow/cmd/mkpj   
+$ bazel-bin/prow/cmd/mkpj/mkpj --config-path ~/istio/test-infra/prow/config.yaml --job test-infra-presubmit | kubectl create -f -
+```
+And give the required information:
```
Comment (chxchx) [inline prow/README.md]: Maybe say "for example" at the beginning to make it clear this is not additional steps the user has to follow

File: prow/README.md (hunk 2 above)
Comment (yutongz) [reply prow/README.md:165]: This is mainly for our information, not a general education. Plus we don't want random people to connect to prow cluster, should be only us.

**Comment by yutongz** [comment]
PTAL

**Pull request merged**